src/imitation/scripts/config/train_rl.py

```diff
@@ -44,6 +46,16 @@ def default_end_cond(rollout_save_n_timesteps, rollout_save_n_episodes):
         rollout_save_n_timesteps = 2000  # Min timesteps saved per file, optional.
 
 
+# Default config for CNN Policies
+@train_rl_ex.named_config
+def cnn_policy():
+    train = dict(
+        policy_cls=CnnPolicy,
+        policy_kwargs=dict(features_extractor_class=NatureCNN),
```

Comment:
We should probably remove the config hook in `common/train.py` and then remove `policy_kwargs`.

Should probably also move the named config itself into `common/train.py`.

Reply:
Do you have pointers to where I should update documentation since I'm removing the hook?

Reply:
On reflection I think we basically don't have the script configs documented, which is sad :( So maybe it's OK to not update the docs. Pro tip: the documentation can't be out of date if you don't have it...

Should update the comment in `normalize_disable` at least though as that's now out of date (mentions the now-removed config hook).